rules/integrations/google_workspace/collection_google_workspace_custom_gmail_route_created_or_modified.toml

```
@@ -0,0 +1,73 @@
+[metadata]
+creation_date = "2022/09/13"
+integration = "google_workspace"
+maturity = "production"
+min_stack_comments = "New fields added: required_fields, related_integrations, setup"
+min_stack_version = "8.3.0"
+updated_date = "2022/09/13"
+
+[rule]
+author = ["Elastic"]
+description = """
+Gmail is by default the mail routing service in Google Workspace. By default, all email from current Google Workspace
+users with accounts are routed through a domain's mail server for inbound and outbound mail. Adversaries can add a
+custom e-mail route for outbound mail to route these e-mails to their own inbox of choice for data gathering. This
+allows adversaries to capture sensitive information from e-mail and potential attachments, such as invoices or payment
+documents.
```

```suggestion
Gmail is the default mail routing service in Google Workspace. By default, all email from current Google Workspace
users with accounts are routed through a domain's mail server for inbound and outbound mail. Adversaries can add a
custom e-mail route for outbound mail to route these e-mails to their own inbox of choice for data gathering. This
allows adversaries to capture sensitive information from e-mail and potential attachments, such as invoices or payment
documents.
```

Can you reorder this to put the concise threat up front and the contextual description after. It is great context to have for the description, but having it up front helps with managing multiple rules and alerts

---

Roger that